content/blogs/2023-10-12-schedulers-landscape.md

```diff
@@ -0,0 +1,179 @@
+
+**No Native Support for Modern DevOps Practices:** As a tool that’s been around for a while, OpCon lacks out-of-the-box support for modern DevOps practices like containerization and Infrastructure as Code (IaC), which are becoming industry standards.
+
+While OpCon offers a feature-rich and flexible scheduling solution, it still leans towards traditional IT environments and could benefit from updates to better support modern DevOps practices and cloud-native applications.
+
+## Control-M
+
+[Control-M](https://www.bmc.com/it-solutions/control-m-capabilities.html) is a workload automation software by BMC Software, aimed at integrating various automated processes across multiple platforms and applications. With roots stretching back to mainframe systems, Control-M has evolved to accommodate modern environments, including cloud-based and containerized applications.
+
+### Core Features
+
+**File Transfer Management**: Control-M offers an advanced file transfer system, allowing users to schedule, transfer, and track files across different platforms.
+
+**Batch Processing:** Control-M can manage complex batch workflows, taking care of dependencies, scheduling, and resource allocation.
+
+**Auditing and Compliance**: Control-M has strong auditing features, providing detailed logs and reports to ensure compliance with various industry standards.
+
+**API Capabilities**: Control-M provides a well-documented set of APIs that allow for broad integration across enterprise services. These APIs can integrate with RESTful services, thereby simplifying the development of complex, multi-step workflows…
+
+### Task-Specific Execution in Complex Flows
+
+Control-M offers the ability to execute a specific task within a flow. If a task fails, you don't have to rerun the entire operation; you can simply pinpoint and re-trigger the individual task.
+
+### How it works
+
+Administrators can use a single interface to schedule, monitor, and manage jobs across different platforms. It supports condition-based job flows, meaning jobs can be set up to trigger based on successful or unsuccessful completion of preceding tasks.
+
+### Where Control-M Could Improve
+
+**Legacy Overhead:** Despite upgrades for modern platforms, the tool carries the weight of its legacy systems focus, which can make it less agile compared to more modern, cloud-native solutions.
+
+Control-M, while strong in traditional scheduling, sometimes falls short in offering the same level of flexibility and ease for defining complex, conditional workflows. The tool's user interface and scripting requirements can be clunky when trying to orchestrate intricate job dependencies, conditional logic, and data-driven triggers. This could potentially slow down organizations that require agile responses to data conditions. With more modern solutions, you often get a flexible, DSL-based (Domain Specific Language) approach that allows for easier, quicker, and more robust workflow design and management.
```

Comment:
```suggestion
Control-M, while strong in traditional scheduling, sometimes falls short in offering the same level of flexibility and ease for defining complex, conditional workflows. The tool's user interface and scripting requirements can be clunky when trying to orchestrate complex job dependencies, conditional logic, and data-driven triggers. This can slow down organizations that require agile responses to data conditions. With more modern solutions, you often get a flexible, DSL-based (Domain Specific Language) approach that allows for easier, quicker, and more robust workflow design and management.

Lastly, the tool is prohibitively expensive. The price tag doesn't justify its capabilities compared to alternatives.
```

I think it's worth mentioning the main reason for folks to migrate away from those legacy systems: they are way too expensive for their limited and clunky functionality.